examples/nmt_transformer/README.md

```diff
@@ -0,0 +1,116 @@
+
+## Objectives
+1. Demonstrate how to package a pre-trained Transformer (NMT) models for English-French and English-German translation with generalized custom handler into torch model archive (.mar) file
+2. Demonstrate how to load model archive (.mar) file into TorchServe and run inference.
+
+## Serve the Transformer (NMT) models for English-French/English-German on TorchServe
+
+* To generate the model archive (.mar) file for English-to-French translation model using following command
+
+    ```bash
+    ./create_mar.sh en2fr_model
+    ```
+    By execuing the above script file, the the model archive (.mar) file will be auto generated in "model_store" folder in the same working directory.
+	
+* To generate the model archive (.mar) file for English-to-German translation model using following command
+
+    ```bash
+    ./create_mar.sh en2de_model
+    ```
+    By execuing the above script file, the the model archive (.mar) file will be auto generated in "model_store" folder in the same working directory.
+
+
+* Start the TorchServe using the model archive (.mar) file created in above step
+
+    ```bash
+    torchserve --start --model-store model_store --ts-config config.properties
+    ```
+
+* Use [Management API](https://github.com/pytorch/serve/blob/master/docs/management_api.md#management-api) to register the model with one initial worker   
+	For English-to-French model
+    ```bash
+    curl -X POST "http://localhost:8081/models?initial_workers=1&synchronous=true&url=TransformerEn2Fr.mar"
+    ```
+	For English-to-German model
+	```bash
+	curl -X POST "http://localhost:8081/models?initial_workers=1&synchronous=true&url=TransformerEn2De.mar"
+	```
+    Note:- Our model works only for one worker.
+
+* To get the inference use the following curl command  
+	For English-to-French model
+    ```bash
+    curl http://127.0.0.1:8080/predictions/TransformerEn2Fr -T model_input/sample.txt
+    ```
+	For English-to-German model
+	```bash
+	curl http://127.0.0.1:8080/predictions/TransformerEn2De -T model_input/sample.txt
+	```
+    Here sample.txt contains simple english sentences which are given as input to [Inference API](https://github.com/pytorch/serve/blob/master/docs/inference_api.md#predictions-api). The output of above curl command will be the french translation of sentences present in the sample.txt file.
+
+## Batch Inference with TorchServe using Translation (NMT) model
+
+### TorchServe Model Configuration
+
+To configure TorchServe to use the batching feature, provide the batch configuration information through "POST /models" API.
```

Comment:
Point to docs/batch_inference_with_ts.md file instead.

Reply:
Done